res/TensorFlowLiteRecipes/FullyConnected_I4_001/test.recipe

```diff
@@ -0,0 +1,45 @@
+operand {
+  name: "in"
+  type: INT4
```

Comment:
To fix https://github.com/Samsung/ONE/pull/12754/files#r1525699747

```suggestion
  type: INT8
```

Reply:
ah... we don't support INT8 yet... -_-;